Mage.Sets/src/mage/cards/l/LocalizedDestruction.java

```diff
@@ -0,0 +1,94 @@
+package mage.cards.l;
+
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.costs.Cost;
+import mage.abilities.costs.common.PayEnergyCost;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.DestroyAllEffect;
+import mage.abilities.effects.common.continuous.GainAbilityControlledEffect;
+import mage.abilities.effects.common.counter.GetEnergyCountersControllerEffect;
+import mage.abilities.keyword.IndestructibleAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.counters.CounterType;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.filter.predicate.mageobject.PowerPredicate;
+import mage.game.Game;
+import mage.players.Player;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class LocalizedDestruction extends CardImpl {
+
+    public LocalizedDestruction(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.SORCERY }, "{3}{W}{W}");
+
+        // You get {E}, then you may pay one or more {E}. If you do, each creature you control with power equal to the amount of paid this way gains indestructible until end of turn.
+
+        // You get {E}
+        this.getSpellAbility().addEffect(new GetEnergyCountersControllerEffect(1));
+
+        // then you may pay one or more {E}. If you do, each creature you control with power equal to the amount of paid this way gains indestructible until end of turn.
+        this.getSpellAbility().addEffect(new LocalizedDestructionEffect());
+
+        // Destroy all creatures.
+        this.getSpellAbility().addEffect(new DestroyAllEffect(StaticFilters.FILTER_PERMANENT_CREATURES));
+    }
+
+    private LocalizedDestruction(final LocalizedDestruction card) {
+        super(card);
+    }
+
+    @Override
+    public LocalizedDestruction copy() {
+        return new LocalizedDestruction(this);
+    }
+}
+
+class LocalizedDestructionEffect extends OneShotEffect {
+
+    LocalizedDestructionEffect() {
+        super(Outcome.AddAbility);
+        this.staticText = "then you may pay one or more {E}. If you do, each creature you control with power equal to the amount of paid this way gains indestructible until end of turn";
+    }
+
+    private LocalizedDestructionEffect(final LocalizedDestructionEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public LocalizedDestructionEffect copy() {
+        return new LocalizedDestructionEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        if (controller == null) {
+            return false;
+        }
+
```

Comment:
Missing possibility to say no to "pay one or more {E}".
In which case there should be an early return true and effect (well until the `DestroyAllEffect` is applied)

The smoothest way is probably to start the getAmount at 0, then check if it is 1 or greater (if not, `return true`)

Reply:
I did it the same was as with [[Aether Refinery]], asking if they want to pay 1 or more then giving the option for amount if they want to

Reply:
I think you are still missing a check on if the player has at least one energy to spend.

Try the card with a [[Vorinclex, Monstrous Raider]] on the opposite side.

Reply:
Well you still are allowed to not pay if you have energy. You just removed that usecase.